### PR TITLE
feat(drivers): GLCDC support for RX72 family

### PR DESCRIPTION
**Pull request by becseya**
This PR extends the GLCDC driver for the RX72 family

### CODE REVIEW

File: src/drivers/display/renesas_glcdc/lv_renesas_glcdc.c [hunk 1]
```diff
@@ -3,14 +3,38 @@
...
+ *********************/
+
+#ifdef _RENESAS_RA_
+    #define USE_FREE_RTOS (BSP_CFG_RTOS == 2)
+#else // RX with SMC code generation
+    #ifndef _RENESAS_RX_
+        #define _RENESAS_RX_ 1
+    #endif
+    #define USE_FREE_RTOS 1
+    #define BYTES_PER_PIXEL 2
```
Comment (liamHowatt) [inline src/drivers/display/renesas_glcdc/lv_renesas_glcdc.c]: Maybe define this in all cases and use it where you have `w * 2`?

**Review by liamHowatt — DISMISSED** [review]
Nice! I really couldn't spot anything. Driver code is allowed to be a bit flexible. I understand this is high-ish priority so I approve.

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Sphinx
+alabaster
```
Comment (liamHowatt) [inline docs/requirements.txt]: Please also add `sphinx-design`.

**Review by liamHowatt — COMMENTED** [review]
Great docs! When I try building the docs from a fresh venv, I needed to add `sphinx-design` to the `requirements.txt` so please add that. Soon we might add a check to the PR CI so we catch this sort of thing before merging :P

Also, if you try building the docs locally, you'll want to merge/rebase master to get the fix for a docs building issue.

File: docs/requirements.txt (hunk 2 above)
Comment (becseya) [reply docs/requirements.txt]: Nice catch. `alabaster` is not needed actually, it probably got mixed up while looking proper package. Did a check with a clean venv.

I was under the assumption the CI does build the docs.

**Review by liamHowatt — APPROVED** [review]

**Comment by liamHowatt** [comment]
Thanks!

**Review by kisvegabor — APPROVED** [review]
Thank you!

**Pull request merged**